config/config-default/core.extension.yml

```diff
@@ -105,6 +106,7 @@ module:
   search: 0
   search_api: 0
   search_api_db: 0
+  sendgrid_integration: 0
```

Comment:
This line should be removed since `sendgrid` is handled by `config_split`

Reply:
Good catch and a missed merge correction.

Reply:
Corrected core extension. If all looks good, just need to deal with composer.lock conflict.